katas/content/solving_graph_coloring/Common.qs

```diff
@@ -12,18 +12,13 @@ namespace Kata.Verification {
     //  - regular-ish graph with 5 vertices (3-colorable, as shown at https://en.wikipedia.org/wiki/File:3-coloringEx.svg without one vertex)
     //  - 6-vertex graph from https://en.wikipedia.org/wiki/File:3-coloringEx.svg
     function ExampleGraphs() : (Int, (Int, Int)[])[] {
-        return [(3, []),
```

Comment:
This was logically grouped by line. I think we should keep this formatting. And in a few other places where matrix is literally represented.

Reply:
fix, see earlier comment